department-of-veterans-affairs/caseflow
# Hide VLJ name in Hearings from VSO users

Resolves [CASEFLOW-2234](https://vajira.max.gov/browse/CASEFLOW-2234)

### Description
For VSO users the VLJ names should not visible.

### Acceptance Criteria
- [ ] The VLJ name column on the Hearing Schedule is not visible
- [ ] The VLJ name column on the Hearing Schedule is not included in the downloaded CSV
- [ ] The VLJ name is not visible in the upper right of the Daily Docket
- [ ] The VLJ name is not visible in the upper right of the Daily Docket Printed

### Testing Plan
1. Shadow user "BILLIE_VSO"
2. Go to /hearings/schedule the VLJ name column should not be visible.
3. Click on "Download Current View" The VLJ name column should not be visible.
4. Click on any Hearing Day on the list to go to the Daily Docket i.e. hearings/schedule/docket/{id}
5. On the top right corner the VLJ name should not be visible.
6. Click on "Download & Print Page" on the top right corner the VLJ name should not be visible.
